extensions/how-long-to-beat/src/index.tsx

```diff
@@ -53,6 +53,7 @@ function SearchListItem({ searchResult }: { searchResult: HowLongToBeatEntry })
 function useSearch() {
   const [state, setState] = useState<SearchState>({ results: [], isLoading: true });
 
+
```

Comment:
You can fix this by running version 2.7.1 of Prettier.

Reply:
Still looking into the detail page. This is also broken.

Reply:
If you need/want, I can try to find time at noon to help you with this.

Reply:
The author stated the detail page is not working anymore https://github.com/ckatzorke/howlongtobeat/issues/40#issuecomment-1276691009
Only thing we can do is tigger the search instead and build the detail that way

Reply:
We actually use prettier 2.8.8 🙂